finance/proposal-calls/Moore_2021.md

```diff
@@ -0,0 +1,90 @@
+
+### Voting and decision
+Voting is done by adding a "thumbs up" to a dedicated comment that will be added to the project PR discussion once the voting period has opened. Any number of projects can be voted up by a single person. The Interim Finance Committee reserves the right to discount "thumbs up" from github accounts without any contributions to Astropy repositories.
+
+The CoCo and the Finance Committee will select projects following the community up/down votes and taking into account the requirements of the available grant funding (e.g. if none of the highest ranking projects deals with spectroscopy, the money reserved for spectroscopy can be awarded to the highest ranking spectroscopy project, even if that means that other, higher ranked projects won't be executed).
+
+*Rationale: This is similar to how e.g. ESO or HST proposals as awarded. Technically, the TACs suggest a list of proposals for the director, who is free to pick in any way they like. However, the TAC ranking is followed unless there are strong reasons against it (e.g. a scheduling requirement cannot be accommodated). This process is well accepted in the community and thus it makes sense to follow it here.*
+
+## Who will be paid to do work?
+Money can be used for travel, infrastructure, equipment etc, but in many cases, money will be used to pay people to do work for Astropy. In that case, the following approaches can be used:
+
+- Projects can list one or more people who will perform the work and any needed budget to support the work.
+- The project team can alternatively not commit to the work themselves, but promise to organize a job search, accept applications, and select a candidate within 6 months of the project being accepted. Contact details of the selected person are then forwarded to the Interim Finance Committee. The project team is not paid for work time associated with the search.
+
+In either case, the Interim Finance Committee will take care of the formal process of setting up statements of work, signing contracts, and reviewing invoices. Other funding strategies that we have not anticipated will also be considered.
+
+Note: The Interim Finance Committee realizes that this process may lead to a number of parallel, possibly competing searches by different groups of people. We expect all searches to follow the code of conduct with respect to fairness, non-discimination etc. Any violation should be brought the Interim Finance Committee or the Astropy Ombudspersion (confidential@astropy.org). If these procedures are violated, funding for that specific project will not be granted. Projects planning to do a search should include the cost of job listings in their budget.
+
+
+## Template for project suggestion
```

Comment:
I couldn't get a suggestion to do this because it wouldn't recognize which were the "real" backticks... but I suggest sorrounding everything below here in triple-backticks so that someone can copy-and-past it direct from github

Reply:
done.